push/models.py

```diff
@@ -87,7 +87,7 @@ class PushApplication(models.Model):
     user = models.ForeignKey(User, on_delete=models.CASCADE)
     name = models.CharField(max_length=255)
     vapid_key = models.CharField(
-        blank=True, max_length=255,
+        blank=False, max_length=255,
```

Comment:
This (and the corresponding migration) makes the `vapid_key` field required so that Django REST framework serializer automatically validates it's not blank and throws a validation error if it is.